apps/tangle-dapp/types/restake.ts

```diff
@@ -0,0 +1,336 @@
+import type {
+  PalletAssetsAccountStatus,
+  PalletAssetsAssetStatus,
+  PalletAssetsExistenceReason,
+  PalletMultiAssetDelegationDelegatorDelegatorStatus,
+  PalletMultiAssetDelegationOperatorOperatorStatus,
+} from '@polkadot/types/lookup';
+
+import { TransformEnum } from './utils';
+
+/**
+ * A request scheduled to change the operator self-bond.
+ * @name PalletMultiAssetDelegationOperatorOperatorBondLessRequest (737)
+ */
+export type OperatorBondLessRequest = {
+  /**
+   * The amount by which the bond is to be decreased.
+   */
+  readonly amount: bigint;
+
+  /**
+   * The round in which the request was made.
+   */
+  readonly requestTime: number;
+};
+
+/**
+ * Represents a bond for an operator from a delegator.
+ * @name PalletMultiAssetDelegationOperatorDelegatorBond (739)
+ */
+export type OperatorDelegatorBond = {
+  /**
+   * The account ID of the delegator.
+   */
+  readonly delegator: string;
+
+  /**
+   * The amount bonded.
+   */
+  readonly amount: bigint;
+
+  /**
+   * The ID of the bonded asset.
+   */
+  readonly assetId: string;
+};
```

Comment:
Some of these properties might be able to be simplified by making their names a bit more explicit, and thus avoiding using comments:
e.g. `OperatorDelegatorBond.amount` -> `OperatorDelegatorBond.amountBonded`
`OperatorDelegatorBond.delegator` -> `OperatorDelegatorBond.delegatorAccountId`